CONTRIBUTOR-DOCS/01_contributor-guides/04_making-a-pull-request.md

```diff
@@ -0,0 +1,179 @@
+    - [New components](#new-components)
+
+</details>
+
+<!-- Document content (editable) -->
+
+This document outlines our team's expectations and best practices for creating and submitting pull requests for Spectrum Web Components.
+
+## Scoping and planning your PR
+
+### Testing
+
+Quality and stability are important. We require writing tests for any fixes or features you introduce. This helps ensure:
+
+- Bugs don't resurface later.
+- New features work as intended for all users.
+- Overall library reliability remains high.
+
+For 1st-gen testing guidance, see the [1st-gen README.md](../../1st-gen/README.md). Testing guidance for 2nd-gen is forthcoming.
+
+If you're unsure how to write tests for certain parts of the library, don't hesitate to ask maintainers for guidance. We appreciate every effort to keep the code solid!
+
+### Documentation
+
+In addition to well-tested code, documentation is crucial. Whenever you add or change a feature, include documentation for it in the relevant areas:
+
+- **README.md**: Each component has a README within its directory. Ensure your changes are included here. This file is used in our generated documentation site.
+- **Comment annotations**: We use comment-based documentation ([JSDocs](https://jsdoc.app/)) so that references are generated automatically where possible.
+
+Accessible, helpful docs are a huge win for everyone, especially newcomers.
+
+### Code formatting
+
+We rely on automated tools like Prettier, ESLint, and Stylelint to enforce style preferences. Setting up these tools in your editor saves time and prevents minor style conflicts from slowing down reviews.
+
+### Accessibility
+
+Since this project is used by a diverse audience, the accessibility of our product is of utmost importance. Features will be evaluated for inclusivity by:
+
+- The use of semantic markup.
+- Labeled interactive elements with appropriate accordance's.
+- Accounting for appropriate states, such as focus and keyboard navigation, according to [standards](https://www.w3.org/WAI/perspective-videos/keyboard/).
+
+If you're unsure about an accessibility detail, the [Web Accessibility Initiative (WAI) ARIA Practices Guide (APG)](https://www.w3.org/WAI/ARIA/apg/patterns/) is a good place to start. You can also open a discussion or ask in your PR.
+
+---
+
+## Developing your PR
+
+### Branch naming
+
+We use a straightforward branch naming convention:
+
+- `[username]/[short-description]` (e.g., `alex/fix-dropdown-bug`)
+- If referencing a known issue, incorporate the issue number (e.g., `alex/123-fix-dropdown-bug`)
+
+### Changeset requirements
+
+For PRs that add or update a component:
+
+- Must include a changeset to trigger the release train and update the CHANGELOG
+- Changeset type should be one of:
+    - `patch` - for bug fixes only
+    - `minor` - for new components or new APIs in an existing component
+    - `major` - for breaking changes to a component or public library API
```

Comment:
We have a guide on changesets that should move to Controbutor Docs and be linked from here.

See https://github.com/adobe/spectrum-web-components/blob/main/projects/documentation/content/guides/writing-changesets.md